packages/payment/src/services/payment-provider.ts

```diff
@@ -173,6 +175,15 @@ export default class PaymentProviderService {
     return res as Record<string, unknown>
   }
 
+  async processEvent(
```

Comment:
suggestion: Rename `getWebhookActionAndData`

I would consider doing the same with the method in the provider interface.

We are only really processing the event in the payment module service method.